### PR TITLE
Img processing

### LINKED ISSUE #38: Configure ActiveStorage to transform plant images
every image that uploaded must be one size,
investigate where and how to configure it?

This [link](https://dev.to/mikerogers0/resize-images-with-active-storage-in-rails-481n) might be useful

### PR DESCRIPTION
**Pull request by pipen95**
Fixes #38

### CODE REVIEW

File: app/views/plants/_add_plant_img.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+= form_for @plant do |f|
+  = f.label :img
+  = f.file_field :img
+  = f.submit 'Change picture', class: 'btn-primary mtn-2'
```
Comment (uzorjchibuzor) [inline app/views/plants/_add_plant_img.html.haml:4]: We currently have a picture uploader function with motor_admin, I don't think we should open this particular function to all users.

File: app/helpers/plants_helper.rb [hunk 2]
```diff
@@ -3,7 +3,11 @@
 module PlantsHelper
   def plant_img(plant)
     if plant.img.present?
-      image_tag plant.img
+      image_tag plant.img.variant(
+        saver: { quality: 50},
+        convert: "jpg",
+        resize_to_fill: [400, 400]
+      )
```
Comment (uzorjchibuzor) [inline app/helpers/plants_helper.rb:10]: This is the crux of the PR, what is required is the uniformity of image sizes regardless of what is uploaded.

File: app/controllers/plants_controller.rb [hunk 3]
```diff
@@ -28,6 +28,17 @@ def create
...
+    @plant = Plant.friendly.find(params[:id])
+    if @plant.update(plant_params)
+      #success
+      redirect_to @plant
+    else
+      #failure
+    end
+   
+  end
+
```
Comment (uzorjchibuzor) [inline app/controllers/plants_controller.rb:41]: You can remove this action for now, the ability to update should stay with  the admin. and that is available at /motor_admin

**Review by uzorjchibuzor — CHANGES_REQUESTED** [review]
Please look at these comments and remember to run `standardrb --fix` before the last commit so that you can pass the code style check.

File: app/views/plants/_add_plant_img.html.haml (hunk 1 above)
Comment (pipen95) [reply app/views/plants/_add_plant_img.html.haml:4]: Yes the partial reference is commented out in the show page.

File: app/controllers/plants_controller.rb (hunk 3 above)
Comment (pipen95) [reply app/controllers/plants_controller.rb:41]: Ok, no got it ;).

File: app/views/plants/_add_plant_img.html.haml (hunk 1 above)
Comment (uzorjchibuzor) [reply app/views/plants/_add_plant_img.html.haml:4]: I understand you, but we should ensure to keep the codebase clutter free, if the code snippet is not being used, it should be removed, just commenting it out should be discouraged.

**Pull request merged**